samples/model-viewer/src/main/java/io/github/sceneview/sample/modelviewer/MainFragment.kt

```diff
@@ -35,7 +35,11 @@ class MainFragment : Fragment(R.layout.fragment_main) {
                 intensity(50_000f)
             }
 
-            val model = sceneView.modelLoader.loadModel("models/MaterialSuite.glb")!!
+            // Animated Model
+//            val model = sceneView.modelLoader.loadModel("models/MaterialSuite.glb")!!
+            // No Animation Model
+            val model = sceneView.modelLoader.loadModel("models/BoomBox.glb")!!
+
```

Comment:
Use this to cycle through an animated and non-animated model.
Note: This change and the BoomBox.glb will be dropped before merging.